bin/ch-run.c

```diff
@@ -57,9 +57,11 @@ const struct argp_option options[] = {
    { "join-pid",       -5, "PID",  0, "join a namespace using a PID" },
    { "join-ct",        -3, "N",    0, "number of join peers (implies --join)" },
    { "join-tag",       -4, "TAG",  0, "label for peer group (implies --join)" },
+   { "logging-test",  -17, "FAIL", OPTION_ARG_OPTIONAL, "test log levels" },
```

Comment:
I wonder if this would be more future-proof as something like:
```suggestion
   { "test",  -17, "TEST", 0, "do test TEST" },
```
with valid `TEST` being `log` and `log-fail`?